src/components/Footer.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import {
+  Address,
+  Footer as FooterUSWDS,
+  FooterNav,
+  Grid,
+  Logo,
+  SocialLinks,
+} from "@trussworks/react-uswds";
+import { useTranslation } from "react-i18next";
+import Link from "../components/Link";
+
+import logoImg from "../../public/static/favicon.ico";
+
+function Footer() {
+  const { t } = useTranslation();
+  const primaryLinks = ["#", "#", "#", "#"];
+
+  return (
+    <FooterUSWDS
+      size="medium"
+      // returnToTop={returnToTop}
+      primary={
+        <FooterNav
+          size="medium"
+          links={Array.from({ length: 4 }, (_, i) => {
+            return (
+              <Link className="usa-footer__primary-link" to={primaryLinks[i]}>
+                {t("footer.primary")[i]}
+              </Link>
+            );
+          })}
+        />
+      }
+      secondary={
+        <Grid row gap>
+          <Logo
+            size="medium"
+            image={
+              <img
+                className="usa-footer__logo-img"
+                // alt={t('')}
+                src={logoImg}
+              />
+            }
+            heading={
+              <p className="usa-footer__logo-heading">
+                {t("footer.secondary.heading.logo")}
+              </p>
+            }
+          />
+          <Grid className="usa-footer__contact-links" mobileLg={{ col: 6 }}>
+            <SocialLinks
```

Comment:
Instead of hardcoding in each of these social media links, socialLinks can be an array in the translation file and you can map through the array here. I guess you'd need to do some string interpolation in the classnames

Reply:
So if I do that, would class names like `usa-social-link--facebook` be in the translation.json file? Where would be the best place to keep classes like these?

Reply:
No, I don't think you need to store the whole class name. I was thinking when you map through the array of social media links it would be something along these lines, with each element being a `socialLink`:

```
<Link
          key={socialLink.name}
           className=`usa-social-link usa-social-link--${socialLink.name}`
            to="#"
   >
```

Reply:
Hard coding is gone, thanks for the helpful code. Check out the newest commit.